pkg/lib/client/openshift/client.go

```diff
@@ -0,0 +1,69 @@
+package ocp
+
+import (
+	"strconv"
+
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	liberr "github.com/konveyor/forklift-controller/pkg/lib/error"
+	util "github.com/konveyor/forklift-controller/pkg/lib/util"
+	core "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const (
+	Token = "token"
```

Comment:
having it in the api package also serves as some kind of documentation of what setting we expect to have in specified secrets, why moving it here?